packages/fleather/lib/util.dart

```diff
@@ -53,3 +53,8 @@ TextDirection getDirectionOfNode(StyledNode node) {
   }
   return TextDirection.ltr;
 }
+
+bool isDataOnlyNewLines(Object data) {
+  if (data is! String) return false;
+  return data.replaceAll('\n', '').isEmpty;
```

Comment:
Don't you think this could be checked quicker and without any memory overhead with a regex?